README.md

```diff
@@ -68,19 +85,22 @@ helm install <YOUR HELM RELEASE NAME> camunda/camunda-platform --set zeebe.prome
 
 You can remove these charts by running:
 
-```
+```sh
 helm uninstall <YOUR HELM RELEASE NAME>
 ```
+> **Note**
```

Comment:
:+1: Looks nice, wasn't aware that github makes this prettier

Reply:
yeah it's a bit recent afaik (added in 2022 probably)